wp-content/themes/pub/wporg-learn-2024/inc/block-hooks.php

```diff
@@ -106,3 +107,92 @@ function update_course_outline_block_add_aria( $block_content, $block ) {
 	}
 	return $html->get_updated_html();
 }
+
+/**
+ * Replace the text for the lesson quiz notice.
+ *
+ * @param string $block_content The block content.
+ *
+ * @return string
+ */
+function update_lesson_quiz_notice_text( $block_content ) {
+	if ( is_singular( 'lesson' ) && is_quiz_ungraded() ) {
+		$tag_processor = new \WP_HTML_Tag_Processor( $block_content );
+
+		// Hide the text "Awaiting grade" in the quiz notice.
+		if ( $tag_processor->next_tag( array(
+			'tag_name' => 'div',
+			'class_name' => 'sensei-course-theme-lesson-quiz-notice__text',
+		) ) ) {
+			$tag_processor->set_attribute( 'style', 'display:none;' );
```

Comment:
I notice you've used `str_replace` below to add an element. Could you use it here to replace this element with an empty string?